api/src/changeRequests/validateChangeRequest.ts

```diff
@@ -62,6 +64,15 @@ export async function validateChangeRequest(
         return validationResult;
     }
 
+    // check ACL entries for group documents
+    if (changeRequest.doc.type === DocType.Group) {
+        validateAclEntry(changeRequest.doc, new GroupAclEntryDto());
+        return {
```

Comment:
This will always fail validation. I think you need to check the output of the validateAclEntry function?